.vscode/settings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "emeraldwalk.runonsave": {
+    "autoClearConsole": true,
+    "commands": [
+      {
+        "match": "\\.proto$",
+        "cmd": "earthly +proto"
+      },
+      {
+        "match": "\\.mod$",
```

Comment:
Interesting.

```suggestion
        "match": "^go.mod$",
```

Reply:
`^` wildcard is not working in this extension for some reason, so closest I can get is `go\\.mod$` which will match `*go.mod`, I guess it good enough